tests/helper/helper_cli.go

```diff
@@ -13,3 +13,16 @@ type CliRunner interface {
 	WaitAndCheckForExistence(resourceType, namespace string, timeoutMinutes int) bool
 	GetServices(namespace string) string
 }
+
+// NewCliRunner initializes new NewCliRunner with the appropriate platform
+func NewCliRunner(cli string) CliRunner {
+	var clirunner CliRunner
+	switch cli {
+	case "oc":
```

Comment:
`kubectlRunner` should also be considered as case.

Reply:
this function is being removed like you suggested here https://github.com/openshift/odo/pull/3075#discussion_r429518183